packages/backend/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
there is a compilerOption so that we don't have to use import * as 
https://stackoverflow.com/questions/56238356/understanding-esmoduleinterop-in-tsconfig-file